Libraries/Image/RCTImageLoader.m

```diff
@@ -74,8 +74,7 @@ + (void)loadImageWithTag:(NSString *)imageTag callback:(void (^)(NSError *error,
           // doesn't spike the memory up during the process.
           @autoreleasepool {
             ALAssetRepresentation *representation = [asset defaultRepresentation];
-            ALAssetOrientation orientation = [representation orientation];
```

Comment:
Turns out this fix doesn't solve my issue here #1845 any ideas @ide its blocking a release for us.